packages/docs/concepts/lowdefy-schema.yaml

```diff
@@ -24,202 +24,274 @@ _ref:
-            - `links: object[]`: An array of `MenuLink` or `MenuGroup` objects that are should be grouped together in the group.
-
-            ###### Menus example:
-            ```yaml
-            lowdefy: LOWDEFY_VERSION
-            menus:
-              - id: default
-                links:
-                  - id: page1
-                    type: MenuLink
-                    pageId: page1
-                    properties:
-                      title: Page 1
-                      icon: AiOutlineFile
-                  - id: top-group
-                    properties:
-                      title: Group
-                      icon: AiOutlineGroup
-                    links:
-                      - id: page2
-                        type: MenuLink
-                        pageId: page2 # pageId will be used as link title
-                      - id: external
-                        type: MenuLink
-                        url: https://external.com
-                        properties:
-                          title: External site
-                      - id: nested-group
-                        type: MenuGroup
-                        properties:
-                          title: Nested Group
-                        links:
-                          - id: page3
-                            type: MenuLink
-                            pageId: page3
-                            properties:
-                              title: Page 3
-              - id: page-1-and-3
-                links:
-                  - id: page1
-                    type: MenuLink
-                    pageId: page1
-                    properties:
-                      title: Page 1
-                      icon: AiOutlineFile
-                  - id: page3
-                    type: MenuLink
-                    pageId: page3
-                    properties:
-                      title: Page 3
-                      icon: AiOutlineControl
-            ```
+            See more about how menu objects are used and defined [here](/menus).
 
             # Pages
 
             Pages in a Lowdefy app are actually just blocks, the building blocks of a Lowdefy app, with a few extra restrictions and features.
 
             Each page should have an `id` that is unique among all pages in the app. Each page is served with the `pageId` as the url route. That is, if you create a page with id `page1`, it will be served at `domain.com/page1`.
 
-            Page blocks should be of category `context`, in order to create a context for the page. This means they always have the context initialization events like `onInit` to initialize the page.
-
             If `properties.title` is set on a page block, the title will be set as the page title (This is the title displayed on the tabs in your browser).
```

Comment:
Please add a TODO that this will change once we add the `head` configuration option on pages, and that we should discuss that here.